app/views/users/show.html.erb

```diff
@@ -21,7 +21,15 @@
       <h3> <%= link_to "Post", user_post_path(@user, post) %> <%= index+1 %> </h2>
       <p><%= post.title %></p>
       <p class='count'>Comments: <%= post.comments.count %> | Likes: <%= post.likes.count %></p>
+      <div class="delete-post">
+        <% if can? :destroy , post %>
+          <%= form_with(url: user_post_path(@user, post), method: :delete) do |form| %>
+            <%= form.submit 'Delete post', data: { confirm: 'Are you sure?' }, class: "delete-button btn" %>
+          <% end %>
+        <% end %>
+      </div>
     </div>
+    
   <% end %>
   <%= link_to "See all posts",user_posts_path(@user), class: "button" %> 
```

Comment:
### OPTIONAL
 
 - Good job so far, when I click the see all post I see this error ⬇️ 
![Screenshot_243](https://github.com/Obimbo07/tdn/assets/69638013/def736cd-cf7d-416c-8da9-c8939f88abd5)

- [ ] You can update your code to show a popup of `no posts yet` and add a button for users to create a new post,  on each user's profile 👍🏿